app/views/layouts/_google_tag_manager.html.erb

```diff
@@ -0,0 +1,7 @@
+<% if ENV['GOOGLE_TAG_MANAGER_TRACKING_ID'].present? %>
+  <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':
+  new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],
+  j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src=
+  'https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);
+  })(window,document,'script','dataLayer', '<%= ENV['GOOGLE_TAG_MANAGER_TRACKING_ID'] %>');</script>
```

Comment:
Here you should be using `Rails.configuration.x.google_tag_manager_tracking_id` rather than called `ENV['GOOGLE_TAG_MANAGER_TRACKING_ID']` directly.